frontend/javascripts/viewer/geometries/plane.ts

```diff
@@ -163,8 +160,12 @@ class Plane {
     }
     this.lastScaleFactors[0] = xFactor;
     this.lastScaleFactors[1] = yFactor;
-    // Account for the dataset scale to match one world space coordinate to one dataset scale unit.
-    const scaleVector: Vector3 = V3.multiply([xFactor, yFactor, 1], this.datasetScaleFactor);
+    // Scale to base voxel space which is the same coordinate space the cameras use
+    const baseVoxelUnit = getBaseVoxelInUnit(this.datasetScaleFactor);
+    const scaleVector: Vector3 = V3.multiply(
+      [xFactor, yFactor, 1],
+      [baseVoxelUnit, baseVoxelUnit, baseVoxelUnit],
+    );
```

Comment:
```suggestion
    const scaleVector: Vector3 = V3.scale(
      [xFactor, yFactor, 1],
      baseVoxelUnit,
    );
```

 + formatting probably